build/edge/README_zh.md

```diff
@@ -13,16 +13,19 @@
 
   以下参数如果不用修改则无需设置
 
-  | 参数名称        | 默认值                            | 备注                     |
-  | --------------- | --------------------------------- | ------------------------ |
-  | cloudhub        | 0.0.0.0:10000                     |                          |
-  | edgename        | edge-node                         |                          |
-  | edge_core_image | kubeedge/edgecore:latest          |                          |
-  | arch            | amd64                             | 可选值：amd64 \| arm64v8 |
-  | qemu_arch       | x86_64                            | 可选值：x86_64 \| aarch  |
-  | certpath        | /etc/kubeedge/certs               |                          |
-  | certfile        | /etc/kubeedge/certs/edge.crt      |                          |
-  | keyfile         | /etc/kubeedge/certs/edge.key      |                          |
+  | 参数名称            | 默认值                       | 备注                     |
+  | ------------------- | ---------------------------- | ------------------------ |
+  | cloudhub            | 0.0.0.0:10000                |                          |
+  | edgename            | edge-node                    |                          |
+  | edge_core_image     | kubeedge/edgecore:latest     |                          |
+  | arch                | amd64                        | 可选值：amd64 \| arm64v8 |
+  | qemu_arch           | x86_64                       | 可选值：x86_64 \| aarch  |
+  | certpath            | /etc/kubeedge/certs          |                          |
+  | certfile            | /etc/kubeedge/certs/edge.crt |                          |
+  | keyfile             | /etc/kubeedge/certs/edge.key |                          |
```

Comment:
duplicated lines below?

Reply:
Sorry, I didn't understand.

Reply:
@zhanghongtong , we have 4 lines with keyfile

Reply:
@zhanghongtong Please resovle it.

Reply:
@m1093782566 Sorry, I am negligent, it has been fixed.